### PR TITLE
Use KDE 6.8 runtime

### PR DESCRIPTION
**Pull request by Angxddeep**
As the new stable version of KDE Runtime is released. This PR updates the runtime to KDE 6.7. Also updates com.riverbankcomputing.PyQt.BaseApp to 6.7 too

### CODE REVIEW

**Comment by AsciiWolf** [comment]
Thanks for your PR! This however needs further testing before merging considering how important this Flatpak is.

And even though it builds fine, it does not seem to run:

```
$ flatpak run org.torproject.torbrowser-launcher//test
Traceback (most recent call last):
  File "/app/bin/torbrowser-launcher", line 30, in <module>
    import torbrowser_launcher
  File "/app/lib/python3.11/site-packages/torbrowser_launcher/__init__.py", line 35, in <module>
    from PyQt5 import QtCore, QtWidgets
ModuleNotFoundError: No module named 'PyQt5'
```

Sadly, torbrowser-launcher is far from being able to run on PyQt6, so it needs PyQt5 bundled.

I am marking this PR as a draft for now.

**Comment by AsciiWolf** [comment]
If possible, please update this PR to use the 6.8 runtime instead. Thanks!

**Comment by Angxddeep** [comment]
> If possible, please update this PR to use the 6.8 runtime instead. Thanks!

com.riverbankcomputing.PyQt.BaseApp hasn't updated their base version to 6.8, although I have updated runtime version to 6.8

**Comment by AsciiWolf** [comment]
> com.riverbankcomputing.PyQt.BaseApp hasn't updated their base version to 6.8, therefore I have only updated runtime version to 6.8

https://github.com/flathub/com.riverbankcomputing.PyQt.BaseApp/issues/386

**Comment by AsciiWolf** [comment]
```
    post-install:
      - mv /app/lib/python3.11/site-packages/gpg-1.23.2-*.egg/gpg /app/lib/python3.11/site-packages/
      - mv /app/lib/python3.11/site-packages/gpg-1.23.2-*.egg/EGG-INFO /app/lib/python3.11/site-packages/gpg-1.23.2-py3.10.egg-info
      - rm -r /app/lib/python3.11/site-packages/gpg-1.23.2-*.egg
```

This probably needs to be changed to python3.13.

**Comment by Angxddeep** [comment]
> ```
>     post-install:
>       - mv /app/lib/python3.11/site-packages/gpg-1.23.2-*.egg/gpg /app/lib/python3.11/site-packages/
>       - mv /app/lib/python3.11/site-packages/gpg-1.23.2-*.egg/EGG-INFO /app/lib/python3.11/site-packages/gpg-1.23.2-py3.10.egg-info
>       - rm -r /app/lib/python3.11/site-packages/gpg-1.23.2-*.egg
> ```
> 
> This probably needs to be changed to python3.13.

done

**Comment by AsciiWolf** [comment]
This update is sadly still blocked by the PyQt BaseApp [not being updated](https://github.com/flathub/com.riverbankcomputing.PyQt.BaseApp/issues/386) to 6.8.

File: org.torproject.torbrowser-launcher.yaml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 base: com.riverbankcomputing.PyQt.BaseApp
-base-version: 5.15-23.08
+base-version: 6.7
 cleanup-commands:
   - /app/cleanup-BaseApp.sh
 build-options:
   env:
     - BASEAPP_REMOVE_WEBENGINE=1
-separate-locales: false
+separate-locales: false
```
Comment (AsciiWolf) [inline org.torproject.torbrowser-launcher.yaml:12]: Please remove the extra space character at the end. It is harmless, but triggers my OCD. :-)) Thanks!

File: org.torproject.torbrowser-launcher.yaml [hunk 2]
```diff
@@ -1,15 +1,15 @@
 app-id: org.torproject.torbrowser-launcher
 runtime: org.kde.Platform
-runtime-version: 5.15-23.08
+runtime-version: 6.8
 sdk: org.kde.Sdk
 base: com.riverbankcomputing.PyQt.BaseApp
-base-version: 5.15-23.08
+base-version: 6.7
```
Comment (AsciiWolf) [inline org.torproject.torbrowser-launcher.yaml:6]: FYI PyQt 6.8 should be released in a week or two (after the Qt 6.8.1 release). Its Flatpak branch will be added afterwards).

**Comment by AsciiWolf** [comment]
Oops, I just realized I forgot about this: https://github.com/flathub/org.torproject.torbrowser-launcher/pull/63#issuecomment-2391172829

On second thought, it will probably be better to update the runtime to `5.15-24.08` that is based on Qt5 and won't require manually bundling Qt5/PyQt5.

**Comment by AsciiWolf** [comment]
Closing in favor of #74.

File: org.torproject.torbrowser-launcher.yaml (hunk 1 above)
Comment (Angxddeep) [reply org.torproject.torbrowser-launcher.yaml:12]: Yeah nowadays, I get a lot of people who have that. :)

**Pull request closed without merging**